.github/workflows/docs-workflow.yml

```diff
@@ -118,3 +118,17 @@ jobs:
             echo "Found broken symlinks: $BROKEN_SYMLINKS"
             exit 1
           fi
+
+
+  check-markdown-links:
+    needs: path_filter
+    if: ${{ needs.path_filter.outputs.should_not_skip == 'true' }}
+    name: Check for broken URLs in markdown files
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8 # pin@v3.1.0
+      - uses: gaurav-nelson/github-action-markdown-link-check@5c5dfc0ac2e225883c0e5f03a85311ec2830d368 # pin@v1
+        with:
+          use-verbose-mode: 'yes'
+          config-file: '.github/workflows/markdown_link_check_config.json'
```

Comment:
I think the config file should be moved to `.github/workflows/config/`

Reply:
Moved